EF-Adapter.UnitTest/packages.config

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.CodeCoverage" version="16.6.1" targetFramework="net45" />
+  <package id="Microsoft.NET.Test.Sdk" version="16.6.1" targetFramework="net45" />
+  <package id="MSTest.TestAdapter" version="2.1.1" targetFramework="net45" />
+  <package id="MSTest.TestFramework" version="2.1.1" targetFramework="net45" />
+  <package id="xunit" version="2.4.1" targetFramework="net45" />
```

Comment:
xunit

Reply:
Yes.You are right. Xunit test mistakenly added.I will uninstall them and update PR.

Reply:
@hsluoyz  Find the updated PR.Thank you